doc/site/changelogs/running-changelog.markdown

```diff
@@ -94,6 +94,16 @@ So far contains three vars:
 
 More will be added in the future as new features are added.
 
+### Lua language server support
```

Comment:
Running changelog is for game-devs, so references to specific engine branches, engine file paths and non-gamedev-facing technologies aren't relevant. I'm not sure where to put these instead since nobody seems to actually maintain a more technical, engine-side changelog. Perhaps a bullet point linking to the other article is fine.

An explanation on what exactly is lost with "a regression in docs quality" would be good.

Reply:
Okay, happy to just drop the engine-specifics entirely then. Sound good?

Reply:
Would be ideal if the knowledge stayed somewhere, just not in this specific file. If you don't want to spend more time on it IMO keep as-is, I can clean it up myself later.

Reply:
Sorry, I got confused and thought all your comments were on the changelog. Yes I'll reorganize the article to put game-dev facing info up top.

Reply:
I've removed the engine specific stuff from the changelog here: 968c97b9b6e373f7ae37474717f1654434169d75

Reply:
FYI I am likely to tackle the docs improvements very soon. Hopefully before the next release. Conveniently someone just released a tool that does a much better job: https://github.com/LuaLS/lua-language-server/issues/3054#issuecomment-2629331945

Pls resolve if you're satisfied with the change.